app/templates/dashboard/contributors/show.hbs

```diff
@@ -0,0 +1,36 @@
+<section id="contributor-profile">
+  <header>
+    {{user-avatar contributor=model size="medium"}}
+  </header>
+
+  <div class="content">
+    <h2>{{model.name}}</h2>
+    <p class="role">{{roleName}}</p>
+
+    <ul class="external-accounts">
+      {{#each model.accounts as |account|}}
+        <li>
+          {{external-account-link account=account}}
+        </li>
+      {{/each}}
+      {{#if model.url}}
+        <li>
+          <a href={{model.url}} target="_blank" rel="noopener" title="Website">
+            <span class="site">Web</span>
+            {{icon-web-globe}}
+          </a>
+        </li>
+      {{/if}}
+    </ul>
+  </div>
+
+  <div class="actions">
+    <p>
+      <a href="https://rinkeby.etherscan.io/address/{{model.account}}" class="button small" target="_blank" rel="noopener">Inspect Ethereum transactions</a>
+      {{#if model.ipfsHash}}
+        <a href="https://ipfs.io/ipfs/{{model.ipfsHash}}" class="button small" target="_blank" rel="noopener">Inspect IPFS profile</a>
```

Comment:
we could use the `gatewayUrl` here

Reply:
That was actually intentional, for two reasons:

1.) Have it be an independent source
2.) Have stuff be cached on ipfs.io so more nodes can distribute it